.github/workflows/ci.yaml

```diff
@@ -38,13 +38,25 @@ jobs:
       - name: Save artifact
```

Comment:
task name: Save radar artifact
artifact-name: radar-${{ github.run_id }}

But!
 As far as we don't need radar static for `test` job, we could just invoke `yarn generate` as a part of `radar-publish` step.